### PR TITLE
DAC6-413 - Add remove functionality for taxpayer

### PR DESCRIPTION
**Pull request by soljeggers**

### CODE REVIEW

File: app/controllers/enterprises/SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController.scala [hunk 1]
```diff
@@ -100,16 +100,16 @@ class SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController @Inject()(
 
   private def reporterToCheckbox(ua: UserAnswers, id: Int): Seq[Checkboxes.Checkbox] =  {
     (ua.get(RoleInArrangementPage, id), ua.get(ReporterIndividualNamePage, id), ua.get(ReporterOrganisationNamePage, id)) match {
-      case (Some(RoleInArrangement.Taxpayer), Some(individualName), None) => Seq(Checkboxes.Checkbox(label = Literal(individualName.displayName), value = s"${individualName.displayName}"))
-      case (Some(RoleInArrangement.Taxpayer), None, Some(organisationName)) => Seq(Checkboxes.Checkbox(label = Literal(organisationName), value = s"${organisationName}"))
+      case (Some(RoleInArrangement.Taxpayer), Some(individualName), None) => Seq(Checkboxes.Checkbox(label = Literal(individualName.displayName), value = "individual-reporter"))
```
Comment (ers2015112) [inline app/controllers/enterprises/SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController.scala:103]: Does this have to be a hard coded string?

File: app/controllers/enterprises/SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController.scala [hunk 2]
```diff
@@ -100,16 +100,16 @@ class SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController @Inject()(
...
+      case (Some(RoleInArrangement.Taxpayer), None, Some(organisationName)) => Seq(Checkboxes.Checkbox(label = Literal(organisationName), value = "organisation-reporter"))
```
Comment (ers2015112) [inline app/controllers/enterprises/SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController.scala:104]: Does this have to be a hard coded string?

File: app/controllers/enterprises/SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController.scala (hunk 1 above)
Comment (soljeggers) [reply app/controllers/enterprises/SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController.scala]: refactored this full section of code now - no longer uses hard coded strings as discussed.

File: app/controllers/enterprises/SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController.scala (hunk 2 above)
Comment (soljeggers) [reply app/controllers/enterprises/SelectAnyTaxpayersThisEnterpriseIsAssociatedWithController.scala]: refactored this full section of code now - no longer uses hard coded strings as discussed.

**Review by ers2015112 — APPROVED** [review]

**Review by ers2015112 — APPROVED** [review]

**Pull request merged**